### ISSUE TITLE
Ability to select/change all module positions 

### ISSUE THREAD

**Issue by steepleian** [lightmaster/Meteobridge-Weather34-Template#31]
Currently the main modules 6,12 and last are selectable. Enhance this ability to select all main modules.

**Comment by weather34** [comment]
probably late december 2019 but here is a sneak preview with all the options you will ever need but im sure it will be never enough. but this works fine just a lot of creating newer modules is the teething part. and it will double up as an APP for mac users :-) 

<img src="https://res.cloudinary.com/brian-underdown/image/upload/v1557736951/december2019_mqrll6.png" width="550" title="weather34 meteobridge template december 2019">

**Comment by steepleian** [comment]
Brilliant, not wanting to wish the rest of the year away, but as a MAC user I will just have to be patient :-)

**Comment by weather34** [comment]
yep patience is a virtue but the prototype works well in terms of using on different devices from mobile phones to desktop + smart tv .but for me internet will be vague until december at the earliest, already maxed  my phone bandwidth limit for the month in a week . think in total there will be 24 options with 24 positions .big advantage is the css file is only 52kb as opposed to existing 112kb aprox. 
the framework is ridiculously slim . just have to sing jingle bells jingle bells jingle all the way oh what fun it is too...
